src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import "./inputWithLabel.css"
+function InputWithLabel({ Labelvalue, placeholder, multiline }) {
```

Comment:
please use the same casing to name all variables `Labelvalue` needs to be either `labelValue` or just `label` as it is self-explanatory